introduction.md

```diff
@@ -0,0 +1 @@
+Hello master, my name is Lukas and I'm mastering GitHub with CodeSchool!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.